packages/ember/tests/reexports_test.js

```diff
@@ -96,7 +96,7 @@ QUnit.module('ember reexports');
   ['watch', 'ember-metal'],
   ['isWatching', 'ember-metal'],
   ['unwatch', 'ember-metal'],
-  ['destroy', 'ember-metal'],
+  ['deleteMeta', 'ember-metal'],
```

Comment:
I think this needs to be:

```js
['destroy', 'ember-metal', 'deleteMeta'],
```

The first arg is `Ember.${path}`.  The second arg is the module it comes from, and the third arg is the export from that module (if different than the first arg).

Reply:
done